content/lua/concepts/Comments/comments.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
the folder should match (all lowercase)

Reply:
Yeah, @qiqi0211 it looks like I was wrong here about the folder changing automatically. If you make another commit from github desktop where you're actually changing something from the comments here, with the revised folder name I think (hope) it will change. Otherwise, I can take a stab at fixing that myself.

Reply:
I tried to rename the folder, but that doesn't seem to work.

Reply:
Don't worry about it, I'll correct it. I believe it's not working because in some cases if it's just a case-change it isn't recognized.